.npmrc

```diff
@@ -1 +1,2 @@
-strict-peer-dependencies=false
\ No newline at end of file
+strict-peer-dependencies=false
+public-hoist-pattern[]=*prisma*
```

Comment:
this was key to the whole thing or webpack complained at build time

see: https://github.com/prisma/prisma/issues/12921